stdlib/public/Cxx/CxxConvertibleToCollection.swift

```diff
@@ -38,13 +38,13 @@ internal func forEachElement<C: CxxConvertibleToCollection>(
 }
 
 extension Array {
-  /// Creates an array containing the elements of a C++ collection.
+  /// Creates an array containing the elements of a C++ container.
   ///
-  /// This initializer copies each element of the C++ collection to a new Swift
+  /// This initializer copies each element of the C++ container to a new Swift
   /// array.
   ///
```

Comment:
This comment "This initializer copies each element of the C++ container to a new Swift array." might be misinterpreted to mean the init is creating a new Swift Array internally.  You could instead say "Initializes the array (or self) by copying every element of the C++ container".

Reply:
Done!